inst/extdata/packages/embarcadero.yaml

```diff
@@ -0,0 +1,32 @@
+name : embarcadero
+title: "Species distribution models with BART"
+version: 1.1
+author: "Carlson C.J."
+maintainer: "Carlson C.J."
```

Comment:
@AMBarbosa , why did you add a pacakge from Carlson C.J. yourself?

Reply:
Because I mentioned the package in the paper, Carlson is not a co-author of the paper, and I use the package a lot in my modelling so I'm quite familiar with its functions. I'm also working on a fork of that package to implement some enhancements, and I plan to submit a pull request soon.